src/views/AfhandelingView.vue

```diff
@@ -323,22 +323,26 @@
             <label :for="'kanaal' + idx" class="utrecht-form-label required"
               >Kanaal</label
             >
-            <select
+            <select 
               :id="'kanaal' + idx"
               v-model="vraag.kanaal"
               class="utrecht-select utrecht-select--html-select"
               @change="setUserChannel"
-              required
+              required          
             >
-              <option>telefoon</option>
-              <option>e-mail</option>
-              <option>contactformulier</option>
-              <option>Twitter</option>
-              <option>Facebook</option>
-              <option>LinkedIn</option>
-              <option>live chat</option>
-              <option>Instagram</option>
-              <option>WhatsApp</option>
+              <template v-if="!kanalenKeuzelijst.data.length">
```

Comment:
Is dit gewenst? Denk dat het dan wel netjes is om consequent te zijn met hoofdletters

Reply:
dit is de oude lijst. die blijft als default. denk dat we tijdelijk de casing aangepast hadden om te matchen met de esuite. nu weer allemaal met hoofdletter